.circleci/config.yml

```diff
@@ -88,24 +88,16 @@ jobs:
     <<: *defaults
     steps:
       - checkout
-      - restore_cache:
-          keys:
-          - v1-dependencies-{{ checksum "yarn.lock" }}
-          # fallback to using the latest cache if no exact match is found
-          - v1-dependencies-
-      - run: yarn
-      - run:
-          name: Write NPM Token to ~/.npmrc
-          command: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
       - run:
-          name: Install dot-json package
-          command: yarn add dot-json --dev
+          name: install
+          command: yarn
       - run:
-          name: Write version to package.json
-          command: $(yarn bin)/dot-json package.json version ${CIRCLE_TAG:1}
+          name: commitlint
+          command: semantic-commitlint -- --ci
       - run:
-          name: Publish to NPM
-          command: npm publish --access=public
+          name: release
+          command: yarn run semantic-release || true
```

Comment:
- I'm missing updated DOCs how it should work now
- NPM_TOKEN is not necessary?
- no caching of dependencies?

Reply:
> NPM_TOKEN is not necessary?

I used this to set it up, so I think it should work :thinking: 
https://github.com/semantic-release/semantic-release/blob/caribou/docs/usage/ci-configuration.md#automatic-setup-with-semantic-release-cli

Reply:
> NPM_TOKEN is not necessary?

I used this to set it up, so I think it should work :thinking: 
https://github.com/semantic-release/semantic-release/blob/caribou/docs/usage/ci-configuration.md#automatic-setup-with-semantic-release-clihttps://github.com/semantic-release/cli#circleci `NPM_TOKEN` has been set up in CircleCI because of this, and I can indeed see it on Circle's dashboard here: https://circleci.com/gh/kiwicom/smart-faq/edit#env-vars